packages/acs-ui-common/src/identifier.ts

```diff
@@ -8,6 +8,13 @@ import {
   isPhoneNumberIdentifier
 } from '@azure/communication-common';
 
+const COMMUNICATION_USER_PREFIX = '8:acs:';
```

Comment:
nit: maybe adding a comment on where to find those prefix

Reply:
Unfortunately, it's only documented internally.